omnibus/config/projects/agent.rb

```diff
@@ -230,6 +230,12 @@
     include_apminject = "true"
   end
 
+  include_procmon = "false"
+  if not windows_arch_i386? and ENV['WINDOWS_DDPROCMON_DRIVER'] and not ENV['WINDOWS_DDPROCMON_DRIVER'].empty?
```

Comment:
:question: not directly related to this PR, but do we still support Windows 32bits?